sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedStartFromNowImpl.java

```diff
@@ -16,12 +16,15 @@ public ChangeFeedStartFromNowImpl() {
 
     @Override
     public void populatePropertyBag() {
-        super.populatePropertyBag();
-
-        setProperty(
-            this,
-            Constants.Properties.CHANGE_FEED_START_FROM_TYPE,
-            ChangeFeedStartFromTypes.NOW);
+        
+            super.populatePropertyBag();
```

Comment:
Does base class method call need to be "synchronized"/single threaded as well?

Reply:
The base class' ChangeFeedStartFromInternal.populatePropertyBag is a no-op as is its baseclass JsonSerializable's populatePropertyBag method.

Reply:
The base class' ChangeFeedStartFromInternal.populatePropertyBag is a no-op as is its baseclass JsonSerializable's populatePropertyBag method.